.github/workflows/build-snap.yml

```
@@ -41,19 +41,27 @@ jobs:
           name: local-${{ needs.build.outputs.snap }}
       - name: test
         run: |
+          set -x
           export COLUMNS=256
           sudo snap install  ${{ needs.build.outputs.snap }} --dangerous
           openstack.sunbeam prepare-node-script | bash -x
           sudo snap connect openstack:juju-bin juju:juju-bin
           sudo snap connect openstack:dot-local-share-juju
           sudo snap connect openstack:dot-config-openstack
           sudo snap connect openstack:dot-local-share-openstack
-          sg snap_daemon "openstack.sunbeam cluster bootstrap --manifest /snap/openstack/current/etc/manifests/edge.yml --accept-defaults --topology single --database single"
+          sg snap_daemon "openstack.sunbeam cluster bootstrap --manifest /snap/openstack/current/etc/manifests/edge.yml --accept-defaults"
```

```suggestion
          sg snap_daemon "openstack.sunbeam cluster bootstrap --manifest /snap/openstack/current/etc/manifests/edge.yml --accept-defaults --topology single --database single"
```

This was done because the runner has more than 32GiB of RAM, this will trigger the run to be in multi mysql mode and we don't want that for the runner.

---

I will recover the line. However, I'm a little bit confused.

> This was done because the runner has more than 32GiB of RAM, this will trigger the run to be in multi mysql mode and we don't want that for the runner.

This is what Sunbeam has in the logic and the tutorial doesn't specify `--topology single --database single`. So you want NOT to deploy multi mysql in the runner, but you want users to deploy multi mysql when there is > 32GiB of RAM.

---

Updated.